app/controllers/home_controller.rb

```diff
@@ -4,7 +4,7 @@ class HomeController < ApplicationController
   def index
     if current_user
       @event = Event.active(early_access: current_user.early_access)
-      @volunteer_roles = @event.volunteer_roles.available_for_user(current_user).all if @event
+      @volunteer_roles = @event.volunteer_roles.available_for_user(current_user).all if @event && !@event.ended?
```

Comment:
This doesn't look like a rubocop autocorrect?

Reply:
It isn't, when we load the homepage if you had an active event, but it was ended, it'd still try load the volunteer roles, which is an extra DB call we don't need, so added to prevent un-needed DB calls

Reply:
Could you remove it from the rubocop commit then please, and add it as part of that change